src/main/java/edu/cornell/kfs/gl/batch/service/impl/CuScrubberProcessImpl.java

```diff
@@ -34,6 +34,7 @@ public class CuScrubberProcessImpl extends ScrubberProcessImpl {
     private static final Logger LOG = LogManager.getLogger(CuScrubberProcessImpl.class);
 
     private static final int CONTINUATION_ACCOUNT_DEPTH_LIMIT = 10;
+    private static final String SCRUBBER_JOB_PLANT_INDEBTEDNESS_COMPONENT = "ScrubberJobPlantFundIndebtedness";

```

Comment:
We may want to ask KualiCo to update the superclass's version of this variable from private to protected.

Reply:
I made a note in the relevant sub-task to report to KualiCo